### PR TITLE
Robots

### PR DESCRIPTION
**Pull request by willronchetti**
- Disable some specific bots we've found to be problematic on FF
- Change error log in inheritance mode computation to an info log as it is spamming sentry when we ingest variant samples without inheritance mode information

### CODE REVIEW

File: src/encoded/inheritance_mode.py [hunk 1]
```diff
@@ -347,7 +347,7 @@ def compute_inheritance_modes(cls, variant_sample, chrom=None):
             cmphet = variant_sample.get("cmphet")
             novoPP = variant_sample.get("novoPP", -1)
         except Exception as e:
-            log.error('Was not able to extract inheritance modes - the required fields do not exist!'
+            log.info('Was not able to extract inheritance modes - the required fields do not exist!'
```
Comment (netsettler) [inline src/encoded/inheritance_mode.py:350]: I don't mind this piggy-backing here, but I don't understand why `log.info` is better...?

Comment (willronchetti) [reply src/encoded/inheritance_mode.py:350]: Sentry will not emit an event for a info log in our current Sentry configuration.

Comment (netsettler) [reply src/encoded/inheritance_mode.py:350]: Oh, I see the explanation in the PR description. Can you put a comment in the code explaining? This seems somehow wrong. Maybe `.debug` is actually the right item here? It feels like not really an info thing.

**Review by netsettler — APPROVED** [review]
My comments are not blocking.

**Pull request merged**